### PR TITLE
fix: add email notifier to argocd-notifier

### LINKED ISSUE #394: email notifier doesn't work
**Describe the bug**
In the [argocd-notifications](https://github.com/argoproj/argo-helm/tree/master/charts/argocd-notifications), setting `email` notifier in values.secret.notifier doesn't work: no email are sent and argocd-notifications-controller pod logs `Failed to process: email is not valid recipient type` error.

**To Reproduce**
1. add email configuration to chart values file, E.G.
```
secret:
  notifiers:
    email:
      host: <host>
      port: <port>
      from: <from address>
      username: <username>
      password: <password>
```

2. Apply configuration

**Expected behavior**
argocd-notifications sends the emails

**Additional context**
Email configuration is documented [here](https://argoproj-labs.github.io/argocd-notifications/services/overview/) and the code that sends email is present as well — [here](https://github.com/argoproj-labs/argocd-notifications/blob/master/notifiers/email.go). I would happy to make a PR to add missing code to helm chart.

### PR DESCRIPTION
**Pull request by dcommisso**
Fixes #394 

Checklist:

* [x] I have update the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have followed the testing instructions in the [contributing guide](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md).
* [x] I have signed the CLA and the build is green.
* [x] I will test my changes again once merged to master and published.

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

File: charts/argocd-notifications/templates/secret.yaml [hunk 1]
```diff
@@ -27,4 +27,13 @@ stringData:
...
 {{- end }}
 {{- end }}
+{{- if .Values.secret.notifiers.email.enabled }}
+    email:
+      host: {{ .Values.secret.notifiers.email.host }}
+      port: {{ .Values.secret.notifiers.email.port }}
+      insecure_skip_verify: {{ .Values.secret.notifiers.email.insecure_skip_verify }}
+      username: {{ .Values.secret.notifiers.email.username }}
+      password: {{ .Values.secret.notifiers.email.password }}
+      from: {{ .Values.secret.notifiers.email.from }}
```
Comment (andyfeller) [inline charts/argocd-notifications/templates/secret.yaml:37]: Should any of these `| quote`?

File: charts/argocd-notifications/values.yaml [hunk 2]
```diff
@@ -69,6 +69,24 @@ secret:
       #     username: username
       #     password: mypassword
 
+    email:
+      # For more information: https://argoproj-labs.github.io/argocd-notifications/services/overview/
```
Comment (andyfeller) [inline charts/argocd-notifications/values.yaml:73]: Thank you for the comments ❤️

File: charts/argocd-notifications/templates/secret.yaml (hunk 1 above)
Comment (dcommisso) [reply charts/argocd-notifications/templates/secret.yaml]: In my tests all was working even without quote but I completely agree with you: it's safer to quote strings. Thank you for pointing that out! I did the change and all is working as expected. I pushed the commit, let me know if there's anything else I can do :-)

**Comment by dcommisso** [comment]
No worries, thanks for accepting my PR!

**Pull request merged**